nncf/tensorflow/graph/metatypes/keras_layers.py

```diff
@@ -538,8 +539,8 @@ class TFLayerNormalizationLayerMetatype(TFLayerMetatype):
 
 
 @KERAS_LAYER_METATYPES.register()
-class TFInputLayerMetatype(TFLayerMetatype):
-    name = 'InputLayer'
+class TFInputLayerMetatype(TFLayerMetatype, InputNoopMetatype):  # must inherit in this exact order so that
```

Comment:
Is it a temporary solution or this is your proposal on how to introduce operator metatypes with the same semantic in the common module?

Reply:
Moved to registry-based grouping rather than inheritance-based